modules/tensor_mechanics/src/materials/TwoStressPlasticity.C

```diff
@@ -4,17 +4,23 @@
 /*          All contents are licensed under LGPL V2.1           */
 /*             See LICENSE for full restrictions                */
 /****************************************************************/
-#include "PQPlasticModel.h"
+#include "TwoStressPlasticity.h"
```

Comment:
TwoInvariantPlasticity vs TwoStressPlasticity.  I prefer the latter, as the yield function does not need to be a function of a stress invariant.  Indeed, there is already the "WeakPlane" plasticity where the yield function is a function of sigma_zz and a combination of sigma_zx and sigma_zy.  We just need the yield function (and flow potential, etc) to be a function of two independent stress parameters.